### PR TITLE
Expiration

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+![Configuring expiration feature on the server](images/expiration.png)
+
+## Eventual consistency
+Once the document is expired it can take up to the delete frequency interval (60 seconds, by default) until the expired documents would acutally be deleted. 
+We do not filter expired documents on load/query/indexing time, so need to a aware that the document might be still there after it expires up to the delete frequency interval timefarme.
+
+## More details
+Internally we track each document with `@expires` property in the metadata, even if the expiration feature it turned off. T
+his way once the expirtaion feature is turned on we can delete all expired documents easily.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown]: turned on -> turned of afterwards ?

File: Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿# Bundle : Expiration
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown]: Please drop "Bundle" - we no longer call it a bundle

Comment (fitzchak) [reply Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown]: Right, removed

File: Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown (hunk 1 above)
Comment (fitzchak) [reply Documentation/4.0/Raven.Documentation.Pages/server/expiration.markdown]: I'll add a section about that support to turn this on and off after the database is already live.

**Pull request merged**